test/srt/test_vision_openai_server.py

```diff
@@ -289,5 +289,26 @@ def test_regex(self):
         assert isinstance(js_obj["number_of_cars"], int)
 
 
+class TestQWen2VLServer(TestOpenAIVisionServer):
+    @classmethod
+    def setUpClass(cls):
+        cls.model = "Qwen/Qwen2-VL-7B-Instruct"
+        cls.base_url = DEFAULT_URL_FOR_TEST
+        cls.api_key = "sk-123456"
+        cls.process = popen_launch_server(
+            cls.model,
+            cls.base_url,
+            timeout=DEFAULT_TIMEOUT_FOR_SERVER_LAUNCH,
+            api_key=cls.api_key,
+            other_args=[
+                "--chat-template",
+                "qwen2-vl",
+                "--disable-radix-cache",
+                "--disable-cuda-graph",
```

Comment:
Are both of them not supported? What is the reason that they are not supported?

If supporting them is hard, we can add some model-specific adjustment code here to set them automatically, so users do not need to manually set them.

https://github.com/sgl-project/sglang/blob/bbd72bfc8609d1c5d8bc9ebb29b9c3b9e218bb90/python/sglang/srt/model_executor/model_runner.py#L94-L107

Reply:
Yes, both of them are still not supported. I will try to make adaptions for them this weekend

Reply:
It support open radix cache now